src/BugsnagUnity/Payload/StackTraceLine.cs

```diff
@@ -92,7 +92,7 @@ public IEnumerator<StackTraceLine> GetEnumerator()
         {
           // if the exception has not come from a stack trace then we need to
           // skip the frames that originate from inside the notifier code base
-          var currentStackFrameIsNotify = stackFrame.MethodName.StartsWith("Bugsnag.Client.Notify");
+          var currentStackFrameIsNotify = stackFrame.MethodName.StartsWith("BugsnagUnity.Client.Notify", StringComparison.InvariantCulture);
```

Comment:
Is there a way to sanitise the stacktrace before passing the value into `StackTraceLine`? Ideally this class shouldn't require knowledge about internal method names elsewhere in the codebase